specs/unified_kernel_image.md

```diff
@@ -159,3 +159,16 @@ that tools staging or consuming UKIs have a common place to store and look for t
 
 The installed UKIs should have a filename `<version format specification>.efi`, i.e. the filename is left to
 implementers but must be valid for comparisons according to the [Version Format Specification](version_format_specification.md).
+
+UKI addons (cmdline PE addons, systemd-sysext extensions for the initrd) built centrally by distributions and
+installed via package manager should be installed into locations depending on whether they are global UKI addons,
+that should be applied to all UKIs installed in the ESP, or whether they should only be applied to a
+single specific UKI.
+
+Global UKI addons should be installed into `/usr/lib/modules/extra.d/`. UKI-specific addons should be installed
+to `/usr/lib/modules/$UNAME/$UKI.efi.extra.d/`, where `$UNAME` is the output of `uname -r` of the kernel included
+in the UKI, `$UKI` is the UKI name just as in `/usr/lib/modules/$UKI.efi`.
```

Comment:
missing and end
```suggestion
in the UKI, and `$UKI` is the UKI name just as in `/usr/lib/modules/$UKI.efi`.
```

My understanding is, that `$UKI.efi` refers to the second paragraph in this section:
> The installed UKIs should have a filename `<version format specification>.efi`, i.e. the filename is left to
implementers but must be valid for comparisons according to the [Version Format Specification](version_format_specification.md).

but the path for that in the first paragraph is `/usr/lib/modules/$UNAME/`, so should this all in all be this?
```suggestion
in the UKI, and `$UKI` is the UKI name just as in `/usr/lib/modules/$UNAME/$UKI.efi`.
```
Maybe this could also be written as 
```suggestion
in the UKI, `$UKI` is the name of the corresponding centrally built UKI with the `.efi` extension stripped.
```

It would probably hammer this home if at the end we added an example, e.g. for kernel version `6.9.1-1-foo` and and UKI name `bar-123` the UKI would be `/usr/lib/modules/6.9.1-1-foo/bar-123.efi` and its addons would go to `/usr/lib/modules/6.9.1-1-foo/bar-123.efi.extra.d/`, e.g. as `/usr/lib/modules/6.9.1-1-foo/bar-123.efi.extra.d/abc-456.addon.efi`. Does that sound correct?

Reply:
Makes sense